packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -30,18 +30,21 @@ export class ChannelService {
   }
 
   public streamCommandOutput(execution: CommandExecution) {
-    this.channel.append(nls.localize('channel_starting_message'));
-    this.channel.appendLine(execution.command.toString());
-    this.channel.appendLine('');
-
-    this.channel.appendLine(execution.command.toCommand());
-
+    this.streamCommandStartStop(execution);
     execution.stderrSubject.subscribe(data =>
       this.channel.append(data.toString())
     );
     execution.stdoutSubject.subscribe(data =>
       this.channel.append(data.toString())
     );
+  }
+
+  public streamCommandStartStop(execution: CommandExecution) {
```

Comment:
@vazexqi I have separated this portion out to suppress the commandOutput. Commands still using streamCommandOutput will still have both the start/ended and the command output.

Reply:
OK. The name is a bit funny `streamCommandStartStop` but since this doesn't affect functionality, we can rename it later with something clearer.